src/Npgsql/TypeMapping/TypeMapperBase.cs

```diff
@@ -141,8 +128,12 @@ public bool UnmapComposite<T>(string? pgName = null, INpgsqlNameTranslator? name
         // TODO: why does ReSharper think `GetCustomAttribute<T>` is non-nullable?
         // ReSharper disable once ConstantConditionalAccessQualifier ConstantNullCoalescingCondition
         static string GetPgName<T>(INpgsqlNameTranslator nameTranslator)
```

Comment:
This can be safely removed and usages replaced by non-generic `GetPgName`.

Reply:
done